src/components/CowCard/CowCard.sass

```diff
@@ -0,0 +1,73 @@
+@import ../../styles/variables.sass
+
+=heartListStyle()
+  .hearts
+    margin-top: .5em
+
+    li
+      display: inline
+      margin: 0 .25em 0 0
+
+.CowCard
+  +heartListStyle()
+
+  .hearts
+    [data-prefix="fas"] // Full heart
+      color: $heart-color
+
+  .card-list & // This specificity is needed to override a global card style
+    +heartListStyle()
+
+  ul.bloodline // This needs the ul specificity to override .card-list styles
+    display: flex
+    margin: 1em 0
+
+    li
+      $indicator-size: 1.25em
+      background: #000
+      border-radius: 50%
+      border: solid 1px #000
+      height: $indicator-size
+      margin-bottom: 0
+      margin-right: 0.5em
+      margin-top: 0
+      width: $indicator-size
+
+      &.blue
+        background: #8ff0f9
```

Comment:
would these colors get updated if the cow colors ever changed? they seem like they should be put into variables somewhere for maintainability